client/src/locales/fr-FR.json

```
@@ -203,7 +203,18 @@
             "tosLink": "https://my.parsec.cloud/assets/pdf/Parsec-CGVU-fr.pdf",
             "and": "et la ",
             "privacyPolicy": "Politique de Confidentialité",
-            "privacyPolicyLink": "https://parsec.cloud/fr/politique-de-confidentialite/"
+            "privacyPolicyLink": "https://parsec.cloud/fr/politique-de-confidentialite/",
+            "update": {
+                "title": "Nouvelles conditions générales",
```

To be consistent with button?
```suggestion
                "title": "Nouvelles conditions générales de vente et d'utilisation",
```

---

We'll stick with "condition générales ~de vente~ d'utilisation"